pom.xml

```diff
@@ -23,6 +23,52 @@
       <url>file:${project.build.directory}/repository</url>
     </repository>
   </distributionManagement>
+
+  <properties>
+    <quarkus.version>0.19.1</quarkus.version>
```

Comment:
Are we really using this old one? we should be on 0.22.x by now

Reply:
That's the one we're using

Reply:
Ok we can upgrade it later. Let’s merge this bad boy